pkg/apis/extensions/types.go

```diff
@@ -127,8 +127,8 @@ type DeploymentSpec struct {
 	// is considered to be failed. The deployment controller will continue to
 	// process failed deployments and a condition with a ProgressDeadlineExceeded
 	// reason will be surfaced in the deployment status. Note that progress will
-	// not be estimated during the time a deployment is paused. This is not set
-	// by default.
+	// not be estimated during the time a deployment is paused. This is set to
+	// the max value of int32 (1<<31 - 1) by default, which means "no deadline".
```

Comment:
Nit: Since this may end up in generated user docs, it might be better to give the explicit value instead (2147483647) so it looks the same way people will see it printed in objects, and so people don't need to be familiar with bit shift syntax.

Reply:
Done.